.github/workflows/examples.yaml

```diff
@@ -0,0 +1,109 @@
+name: examples
+
+on:
+  push:
+    paths:
+    - 'examples/**'
+    - 'packages/**'
```

Comment:
```suggestion
    - 'packages/*/example/**'
    - 'performance_tests/**'
```

Reply:
The examples in `examples/` use paths for bloc packages dependencies. With the proposed version, we'd check changes in `packages/` with that corpus of examples on each commit and PR. With your suggestion my understanding is that we'd not verify the compatibility of library changes against examples, just modifications to examples themselves.

Reply:
OK, I see.

1) Well, when someone does a change to the package, we should build and test only packages. And not all the examples. (**`packages/**` in main.yaml**)
2) And we should build examples only when there is some change to it. (**`performance_tests/**` and `examples/**` in examples.yaml**)
3) A special case is an example in a package, which is using the bloc etc. reference (`../bloc`). But if we add 'packages/**' to the examples build, we would had to do the testing and build everytime something changes, no matter where. So that would not help, because testing alld `examples/` is time consuming. So I would probably suggest moving `packages/*/example` to main CI? **So that would mean adding example test & build to main.yaml**. What do you think @netantho

Reply:
OH, wow. Sorry, this was my bad, sorry for the confusion. I can see now that the examples also use relative path.

For example
```
flutter_bloc:
  path: ../../packages/flutter_bloc
``` 

I thought they are fixed to a specific version. OK, in that case, I guess we have to build everything on any change to the repository?

Oh, I can see that those builds take surprisingly not so much time, so that would be OK I guess? Cc @felangel 

For example:
![image](https://user-images.githubusercontent.com/5287596/73784239-63e6f100-4795-11ea-831f-caf16da39373.png)

Reply:
In that case, we actually could merge those CI files? And create jobs like `packages` (current main.yaml) and `examples`. Simple names, so we don't have to have something crazy like `analyze-and-test-and-build-apk`. 🤣

Reply:
And please, just a nit: could you reorder matrix so folders are first and OS second? That would help to create better previews, now it shows OS first and that not useful. 😄 💙

Reply:
Thanks @tenhobi! Implemented 👍